terraform/shared/shared.tf.json.template.py

```diff
@@ -44,7 +44,7 @@ def conformance_pack(name: str) -> str:
     CloudTrailAlarm(name='api_unauthorized',
                     statistic='Sum',
                     filter_pattern='{($.errorCode="*UnauthorizedOperation") || ($.errorCode="AccessDenied*")}',
-                    threshold=12,
+                    threshold=8,
```

Comment:
A brief comment should explain our reasoning here. How many alarms do we expect, and from what source. Like "Four alarms from config per day with custom delivery_frequency below, and two from SSM at least once per week, or on every reboot of the GitLab instance. Plus a margin of …"